test/integration/default/config_spec.rb

```diff
@@ -147,7 +147,7 @@
     # Match 100.51.198 reverse zone from pillar
     its('content') { should match /^zone\ "100\.51\.198\.in-addr\.arpa"\ {\n\ \ type\ master;\n\ \ file\ "#{zones_directory}\/100\.51\.198\.in-addr\.arpa";\n\ \ \n\ \ notify\ no;\n\};/ }
     # Match logging
-    its('content') { should match /^logging\ \{\n\ \ channel\ "querylog"\ {\n\ \ \ \ file\ "#{log_directory}\/query\.log";\n\ \ \ \ print-time\ yes;\n\ \ \};\n\ \ category\ queries\ \{\ querylog;\ \};\n\};/ }
+    its('content') { should_not match /^logging\ \{\n\ \ channel\ "querylog"\ {\n\ \ \ \ file\ "#{log_directory}\/query\.log";\n\ \ \ \ print-time\ yes;\n\ \ \};\n\ \ category\ queries\ \{\ querylog;\ \};\n\};/ }
```

Comment:
As you're moving the logging config to another file, can you either add some tests to make sure the `logging_config` file/s are created properly, or open an issue so we don't forget to add tests for this? Perhaps I can find some time to add them

Reply:
Okay. Let me rework this a bit...